catalog/ui/catalog-ui-search/src/main/webapp/component/search-settings/search-settings.less

```diff
@@ -31,14 +31,23 @@
     }
   }
 
-  > .editor-properties > button {
+  > .editor-properties > .editor-btn-group > button {
     display: none;
     width: 100%;
   }
 }
 
 @{customElementNamespace}search-settings.show-save {
-  > .editor-properties > button {
+  > .editor-properties > .editor-btn-group > .editor-save {
     display: block;
   }
-}
\ No newline at end of file
+}
+
+@{customElementNamespace}search-settings.save-cancel {
+  > .editor-properties > .editor-btn-group > button {
```

Comment:
Instead of float use inline block for the display.  Position relative shouldn't be necessary here.